### PR TITLE
Allow the documentation to use a specific tag when required

### PR DESCRIPTION
**Pull request by lfades**
If in the future we need documentation changes to be released but a new Next.js is not yet out, this change can allow us to force a specific tag.

Note: could we use Now secrets for this? I don't know if that could work as a dynamic way of changing the tag by starting a new build :thinking: 

Alternatively, this could use a static env variable inside `next.config.js`, but I didn't see the need for it.

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/docs/\[...slug\]**](https://next-site-git-force-tag.zeit.now.sh/docs/[...slug]) |
|:-:|
|<a href="https://next-site-git-force-tag.zeit.now.sh/docs/[...slug]">Screenshot of /docs/[...slug]</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.us-east-1.amazonaws.com/lt_BZ1tBsG786q77OGlb11JDDag_.png">(view full size)</a> |







Commit 0691e83f60f35c06ffe1353baed1eee9646689a2 (https://next-site-git-force-tag.zeit.now.sh).

File: lib/docs/page.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 
+// Default tag if the latest release was not found
+const TAG = 'v9.3.2-canary.4';
+// If a version different from the latest release is required, update TAG with the wanted
+// version and set this to `true`
+const FORCE_TAG = true;
+
+export async function getCurrentTag(tag) {
+  if (FORCE_TAG) return TAG;
+  return tag || getLatestTag();
```
Comment (chibicode) [inline lib/docs/page.js]: Do we need `await` for `getLatestTag()` here?

Comment (lfades) [reply lib/docs/page.js]: No, only if we wanted to handle an error for it there :+1:

Comment (lfades) [reply lib/docs/page.js]: otherwise `return await` is the same of just doing `return` if the function has `async`

Comment (chibicode) [reply lib/docs/page.js]: Good point 👍 https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md

**Pull request merged**